### ISSUE TITLE
Port Group selector broken for Top Interfaces Widget

### ISSUE THREAD

**Issue by dracoling** [librenms/librenms#16715]
### The problem

One of my primary dashboards features a "Top Interfaces" widget with a Port Group selector, today this widget simply shows "Problem with Backend", I recall this working within the past few days. 

After some digging I think that the recent changes in the ports model in #16430 may have broken the sub-query by replacing `port_group_port`.`port_group_id` with `ports`.`port_group_id`

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 24.10.1-39-gb0b82c8b5 (2024-11-14T18:45:10-05:00)
DB Schema | 2024_10_13_162920_add_custom_map_nodeimage_column (307)
PHP       | 8.3.12
Python    | 3.11.2
Database  | MariaDB 10.11.6-MariaDB-0+deb12u1
RRDTool   | 1.7.2
SNMP      | 5.9.3
===========================================

[OK]    Composer Version: 2.8.2
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQL and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrd_dir is writable
[OK]    rrdtool version ok
```

### Anything in the logs that might be useful for us?

```txt
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ports.port_group_id' in 'where clause' (Connection: mysql, SQL: select `port_id`, `device_id`, `ifName`, `ifDescr`, `ifAlias` from `ports` where (`ports`.`deleted` = 0 and `ports`.`disabled` = 0) and `poll_time` > 1731677544 and (`ports`.`deleted` = 0 and `ports`.`ignore` = 0 and `ports`.`disabled` = 0 and `ports`.`ifOperStatus` = up) and exists (select * from `devices` where `ports`.`device_id` = `devices`.`device_id`) and `ports`.`port_id` in (select `port_id` from `port_group_port` where `ports`.`port_group_id` = 1) group by `port_id`, `device_id`, `ifName`, `ifDescr`, `ifAlias` order by SUM(LEAST(ifInOctets_rate, 9223372036854775807) + LEAST(ifOutOctets_rate, 9223372036854775807)) DESC limit 5) {"userId":1,"exception":"[object] (Illuminate\\Database\\QueryException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ports.port_group_id' in 'where clause' (Connection: mysql, SQL: select `port_id`, `device_id`, `ifName`, `ifDescr`, `ifAlias` from `ports` where (`ports`.`deleted` = 0 and `ports`.`disabled` = 0) and `poll_time` > 1731677544 and (`ports`.`deleted` = 0 and `ports`.`ignore` = 0 and `ports`.`disabled` = 0 and `ports`.`ifOperStatus` = up) and exists (select * from `devices` where `ports`.`device_id` = `devices`.`device_id`) and `ports`.`port_id` in (select `port_id` from `port_group_port` where `ports`.`port_group_id` = 1) group by `port_id`, `device_id`, `ifName`, `ifDescr`, `ifAlias` order by SUM(LEAST(ifInOctets_rate, 9223372036854775807) + LEAST(ifOutOctets_rate, 9223372036854775807)) DESC limit 5) at /opt/librenms/vendor/laravel/framework/src/Illuminate/Database/Connection.php:829)
[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ports.port_group_id' in 'where clause' at /opt/librenms/vendor/laravel/framework/src/Illuminate/Database/Connection.php:423)"}
```

**Comment by laf** [comment]
If you can submit a fix then please do so.

**Comment by dracoling** [comment]
I submitted a one line fix in PR #16716 this works in my environment.
